adapters/marsmedia/marsmedia_test.go

```diff
@@ -0,0 +1,11 @@
+package marsmedia
+
+import (
+	"testing"
+
+	"github.com/prebid/prebid-server/adapterstest"
+)
+
+func TestJsonSamples(t *testing.T) {
+	adapterstest.RunJSONBidderTest(t, "marsmediatest", NewMarsmediaBidder("http://bid306.rtbsrv.com/bidder/?bid=f3xtet"))
+}
```

Comment:
I'm getting an error when running the tests. Is this just me?
```
╰─➤  go test
go: finding github.com/prebid/prebid-server/adapterstest latest
# github.com/prebid/prebid-server/adapters/marsmedia
package github.com/prebid/prebid-server/adapters/marsmedia (test)
        imports github.com/prebid/prebid-server/adapterstest: no matching versions for query "latest"
FAIL    github.com/prebid/prebid-server/adapters/marsmedia [setup failed]
```

Reply:
No, i get it also,
what this error? i need get some other version?